src/Relation/Unary.agda

```diff
@@ -161,6 +161,14 @@ syntax IUniversal P = ∀[ P ]
 Decidable : Pred A ℓ → Set _
 Decidable P = ∀ x → Dec (P x)
 
+-- Erasure: A decidable predicate gives rise to another one, more
+-- amenable to η-expansion
+
+⌊_⌋ : {P : Pred A ℓ} → Decidable P → Pred A ℓ
```

Comment:
Hmm I'm a bit worried about this name clashing with `⌊_⌋` in `Relation.Nullary.Decidable.Core`. Are there any other symbols we could use? Maybe `⌞_⌟`?

Reply:
Alternatively the `Core` definition could be deprecated & renamed to `isYes`
(and we could throw in `isNo` too for good measure)?

Reply:
Yup sounds good to me. Sorry for the long reply time!